tests/docs/test_docs.py

```diff
@@ -12,24 +12,28 @@ def path_to_docs(self):
     def test_html(self):
         wd = os.getcwd()
         os.chdir(os.path.sep.join(self.path_to_docs))
-
-        response = subprocess.run(["make", "html-noplot"])
-        self.assertTrue(response.returncode == 0)
-        # response = subprocess.call(["make", "html"], shell=True)  # Needed for local test on Windows
-        # self.assertTrue(response == 0)
+        
+        try:
+            response = subprocess.run(["make", "html-noplot"])
+            self.assertTrue(response.returncode == 0)
+        except:
+            response = subprocess.call(["make", "html"], shell=True)  # Needed for local test on Windows
+            self.assertTrue(response == 0)
 
         os.chdir(wd)
 
     def test_linkcheck(self):
         wd = os.getcwd()
         os.chdir(os.path.sep.join(self.path_to_docs))
-
-        response = subprocess.run(["make", "linkcheck-noplot"])
-        print(response.returncode)
-        self.assertTrue(response.returncode == 0)
-        # response = subprocess.call(["make", "linkcheck"], shell=True)  # Needed for local test on Windows
-        # print(response)
-        # self.assertTrue(response == 0)
+        
+        try:
```

Comment:
Again, same comment as above. Undo this, or figure out if that change is actually necessary.